src/battle_util.c

```diff
@@ -4648,7 +4648,8 @@ u8 AbilityBattleEffects(u8 caseID, u8 battler, u16 ability, u8 special, u16 move
              && (gMultiHitCounter == 0 || gMultiHitCounter == 1)
              && !(GetBattlerAbility(gBattlerAttacker) == ABILITY_SHEER_FORCE && gBattleMoves[gCurrentMove].flags & FLAG_SHEER_FORCE_BOOST)
              && (CanBattlerSwitch(battler) || !(gBattleTypeFlags & BATTLE_TYPE_TRAINER))
-             && !(gBattleTypeFlags & BATTLE_TYPE_ARENA))
+             && !(gBattleTypeFlags & BATTLE_TYPE_ARENA)
+             && !IsLastAliveMonInParty(battler))
```

Comment:
a brand new function is a bit unnecessary here. There are functions like `CountAliveMonsInBattle` and `CountUsablePartyMons` already. I recommend the former.

Reply:
I felt there had to be a function for this, but I couldn't find it when I searched for "alive", which is why I ended up writing `IsLastAliveMonInParty`, lmao.

I erased it and made the check read `CountUsablePartyMons`, because `CountAliveMonsInBattle` didn't look as usable for this. Thank you!